template/Gemfile

```diff
@@ -15,9 +15,9 @@ end
 gem 'bundler', '~> 2.0'
 gem 'rake', '~> 13.0'
 gem 'rspec', '~> 3.0'
-gem 'simplecov', '~> 0.21'
+gem 'simplecov', '~> 0.22.0'
 
-gem 'rubocop', '~> 1.31'
-gem 'rubocop-packaging', '~> 0.5'
-gem 'rubocop-performance', '~> 1.0'
-gem 'rubocop-rspec', '~> 2.20'
+gem 'rubocop', '~> 1.60.0'
+gem 'rubocop-packaging', '~> 0.5.2'
+gem 'rubocop-performance', '~> 1.20.2'
+gem 'rubocop-rspec', '~> 2.26.1'
```

Comment:
These are fine to have this granular, if combined with a bot which updates them.

Reply:
#11 is still open.

Anyway, with such locks for RuboCop there can be a new offense from any plugin or core gem at any moment.

For example, `main` build is OK, but then somebody creates a PR and gets CI fails non-related to the PR.

Reply:
Oh, maybe you meant a bot for a generated library. Yeah, it should be. Anyway, it's worth it to do in such way.

Reply:
#11 is merged now